CONTRIBUTING.md

```diff
@@ -1,46 +1,61 @@
-# CaImAn contributors guide
-CaImAn is an open source project where *everyone* is welcome and encouraged to contribute. We have external contributors from all over the world, and we are always looking for more help. The guide below explains how to contribute to Caiman: if you have questions about the process, please feel free to reach out at [Gitter](https://app.gitter.im/#/room/#agiovann_Constrained_NMF:gitter.im). Everyone typically needs some help at first, so please don't be shy about asking. 
+# CaImAn contributors guide :hammer:
+CaImAn is an open source project where *everyone* is welcome and encouraged to contribute. We have external contributors from all over the world, and we are always looking for more help. This guide explains how to contribute: if you have questions about the process, please feel free to reach out at [GitHub Discussions](https://github.com/flatironinstitute/CaImAn/discussions). Everyone needs help contributing and finds git/GitHub confusing, so please don't be shy about asking. 
```

Comment:
There's some language in here I'd like to trim (everyone finds git/github confusing? That's a bit much) but it's fine.

I don't really want emojis to keep creeping into more of our docs though.